app/jobs/cache/browser_channels/responses_for_prefix.rb

```diff
@@ -21,7 +21,7 @@ def perform(prefix)
   def generate_brotli_encoded_channel_response(prefix:)
     @site_banner_lookups = SiteBannerLookup.where("sha2_base16 LIKE ?", prefix + "%")
     channel_responses = PublishersPb::ChannelResponseList.new
-    @site_banner_lookups.includes(publisher: :uphold_connection).includes(publisher: :paypal_connection).each do |site_banner_lookup|
+    @site_banner_lookups.includes(publisher: [:uphold_connection, :bitflyer_connection, :gemini_connection, :paypal_connection]).each do |site_banner_lookup|
```

Comment:
```suggestion
    @site_banner_lookups.includes(publisher: [:uphold_connection, :bitflyer_connection, :gemini_connection]).each do |site_banner_lookup|
```

Reply:
Good idea! I had initially removed all paypal support from this PR, but reverted it since I wanted to keep the concepts separate. I want to put up a new PR that only removes Paypal support.